content/guides/access-control.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Enable and configure access control
+weight: 1
+description: A tutorial for creating Role-based Access Control in Upbound
+---
+
+{{< hint "important" >}}
+This feature is in alpha and requires Spaces `v1.3.0`.
+{{< /hint >}}
+
+This guide introduces role-based access control (RBAC) in Upbound. RBAC allows you to apply privileges to your Upbound objects and control planes. Organizations using RBAC can manage user roles and privileges to determine the level of access a user can have within their role.
```

Comment:
If we now focus this page on how to enable Kubernetes RBAC, look at the wording in the k8s RBAC definitions. It's all about assigning subjects (users, teams) to roles (which give access to certain objects) through rolebindings. We should try to be somewhat consistent in using that terminology.

We should always specify "which" RBAC we talk about, Kubernetes or Upbound. Otherwise it gets very confusing as these are completely different.